### DISCUSSION TITLE
v3 connection timeout

### DISCUSSION THREAD

**Discussion by jonadis** [mrlt8/docker-wyze-bridge#393] [Q&A]
Greetings,

I had all this working a few months ago on the bench and finally got around to installing the camera in its permanent location.  The WyzeCam v3's firmware has changed and so has the build of wyze-bridge, so I'm not sure if either of those changes had anything to do with this.  In any case, I can't seem to get the bridge to connect to the v3 even though the v3 shows up in the App just fine...

In the logs I just get this over and over...
```
🎬 STARTING 1 OF 3 CAMERAS
2022/05/25 19:22:26 [WyzeBridge] Starting rtsp-simple-server v0.18.4
2022/05/25 19:22:26 [WyzeBridge] 🎉 Connecting to WyzeCam V3 - Chicken on 10.0.70.138 (1/3)
2022/05/25 19:22:58 [WyzeBridge] ⏰ Timed out connecting to Chicken (30s).
2022/05/25 19:23:08 [WyzeBridge] 🎉 Connecting to WyzeCam V3 - Chicken on 10.0.70.138 (1/3)
2022/05/25 19:23:40 [WyzeBridge] ⏰ Timed out connecting to Chicken (30s).
```

That IP is definitely reachable from the docker-host running wyze-bridge, just as it was before.

**Comment by jonadis** [comment]
Update, I figured it out...
I had also turned up a web proxy that's between the docker host and the internet.
The proxy doesn't seem to like some kind of communication between the wyze-bridge and the Wyze cloud...
```
"ProxyDeny: HTTP Invalid Request-Line format"
```
In any case, I just put in a rule to bypass the proxy for the docker-host and now all is well.

**Reply by Kaydeross** [reply]
How to add that rules ? Can you share ?
